Dockerfile

```diff
@@ -13,10 +13,8 @@ RUN make install
 RUN make build-prod
 RUN make prune
 
-FROM registry.access.redhat.com/ubi8/nodejs-14:1
-USER root
-RUN yum -y remove nodejs-nodemon
-RUN yum -y --nobest update
+FROM registry.ci.openshift.org/open-cluster-management/common-nodejs-parent:nodejs-14
+RUN microdnf update
```

Comment:
I think we need to add:
```
RUN ./install-npm.sh
```
https://github.com/open-cluster-management/common-nodejs-parent/blob/6b20341a3d71efdb5e541a2b2b2727a6dcfb71c7/Dockerfile#L11-L14

Reply:
Oh, nevermind--we run `node` directly.

Reply:
Sorry for the back-and-forth between...myself. 😆 

Do we need to update the builder image at the top of the file? If so, we'll need to install NPM.